src/status_im/browser/core.cljs

```diff
@@ -61,9 +73,35 @@
   (let [history-host (http/url-host (try (nth history history-index) (catch js/Error _)))]
     (assoc browser :unsafe? (dependencies/phishing-detect history-host))))
 
+(defn ens? [host]
+  (and (string? host)
+       (string/ends-with? host ".eth")))
```

Comment:
In the future others domain will be available so we will need something more robust.
For now can we make this a utility and share with others part of the codebase? i.e. pretty sure chat already has this somewhere